### ISSUE TITLE
Missing DTD file, corbertura.sourceforge.net is offline

### ISSUE THREAD

**Issue by neilsh** [cobertura/cobertura#366]
corbertura.sourceforge.net is down with the following message:

`Project web is currently offline pending the final migration of its data to our new datacenter.`

This breaks XML files which reference the DTD at http://cobertura.sourceforge.net/xml/coverage-04.dtd, such as those generated by this code: https://github.com/cobertura/cobertura/blob/db3bedf3334d8f35bad7ca3c6f4d777be6a09fc5/cobertura/src/main/java/net/sourceforge/cobertura/reporting/xml/XMLReport.java#L66

Since the project itself has migrated, it may make sense to also move the canonical DTD url to github. Related to this issue from a few years ago: https://github.com/cobertura/cobertura/issues/38

**Comment by raatiniemi** [comment]
Yes, change of the DTD would be much appreciated, it feels like sourceforge is down every other day.
